modules/calendar/app/controllers/calendar/ical_controller.rb

```diff
@@ -28,6 +28,9 @@
 
 module ::Calendar
   class ICalController < ApplicationController
+    # Authentication and authorization is handled within the service.
+    skip_before_action :check_if_login_required
```

Comment:
Just out of curiosity: Why was this not an issue on the pull preview environment?

Reply:
I see!